packages/packages.csv

```diff
@@ -71,7 +71,7 @@ Microsoft.AspNetCore.Mvc.Localization,ship,include
 Microsoft.AspNetCore.Mvc.Razor,ship,include
 Microsoft.AspNetCore.Mvc.Razor.Extensions,ship,include
 Microsoft.AspNetCore.Mvc.RazorPages,ship,include
-Microsoft.AspNetCore.Mvc.Razor.ViewCompilation,ship,exclude
+Microsoft.AspNetCore.Mvc.Razor.ViewCompilation,ship,include
```

Comment:
Marking as ship,include for now since it should be in the metapackage. This functionally does not affect anything right now but we should revisit what these fields mean. ship vs no-ship, include in metapackage vs not, include in cache vs not.